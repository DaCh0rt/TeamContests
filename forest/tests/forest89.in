353 311
-339 6756
5573 -2399
-7485 -498
5175 768
8571 -2364
-7465 -4700
7264 -7713
-4546 5816
2069 5579
5435 -1508
-9983 4710
6240 7088
-8985 6291
-9968 5935
7326 -4219
6925 1986
9388 1290
979 9848
-2887 1484
2652 7904
3582 -4839
-4356 1321
-7453 -1981
-3680 2214
-3540 9749
-1915 8164
-3339 -3739
-4212 3016
3897 3661
8657 -4755
5428 9292
1872 990
-6292 -2652
2541 -1089
7587 -1549
-958 5511
-5592 -9116
4839 -6960
8009 -8526
5340 6340
5160 2647
-2084 8324
5240 2779
-2638 2444
3351 6246
-7910 -9503
4253 -5246
-1836 9040
1858 -9071
-2699 920
4018 -2704
-1237 -5499
-7646 -6137
1910 7645
6596 -7384
6849 3209
9898 895
9544 9755
7717 1939
-9004 -3034
4601 -1875
-8096 -9228
-2413 5236
-8121 4092
-911 4252
-9873 7216
3457 9378
8624 6231
-1794 1726
6855 174
-9216 3508
8976 -1949
-3332 -9915
9788 -4337
-2039 8787
-2893 -5835
6239 -6080
-48 4635
-4410 4699
9770 -7590
866 -4620
4493 4530
9216 -7431
2807 -665
1908 -2327
634 -6501
3788 -7148
7777 8654
-7611 6790
-7090 9731
2783 9804
-1522 7222
7028 94
-9024 9776
2779 7882
-8116 4888
-3729 899
-8085 3598
-5376 8203
1674 1968
-6992 106
-7874 2660
2228 -5906
619 3044
9567 -8302
-6418 9259
4893 -2767
-2641 -8438
-2297 8157
6517 9892
1276 2317
-3584 -5244
-9741 -458
3414 137
-2806 -772
-4382 -3709
-7415 -1752
5749 6463
6400 2656
-1622 5105
7884 -2150
2922 1638
2545 1731
1930 -5027
-6524 9854
8281 -1136
-4294 -7829
599 1056
-9075 -6641
-8557 -9879
-907 -3902
-843 -5594
-6335 2923
9174 -7747
-4258 -4319
-9880 1133
8567 2801
7000 -7288
-7448 4589
4841 8206
-4310 -5167
792 3214
-9440 5256
-1294 6886
-9710 6937
4308 9229
5078 -1991
9252 648
1042 4792
2037 2871
3059 1564
-9922 -1317
-560 392
-3138 2667
2660 8197
3745 -7843
6121 -8633
-1688 -6311
9903 -45
6530 2580
7816 8898
9149 -617
-9214 -521
-9069 6256
-2565 4303
-3082 7939
-2629 7469
5665 1532
-8597 -4423
9269 -4609
-9557 -6040
-2672 9281
-1898 -1562
9116 7805
426 -6261
-967 -5084
-3991 6181
3055 -3996
7474 -2351
535 9820
-3736 -9138
1627 -8619
-1319 6151
1048 -3392
4372 -751
-3264 2075
7343 -9759
-8055 -9751
-6439 -1671
192 -865
5096 7899
-8293 -6835
248 7153
-7925 5467
-1077 3658
-741 -3018
-7404 209
5682 -8173
-4703 734
2334 -3881
-8102 -2780
7568 -3965
4200 -7820
-5621 -4786
-217 -1545
98 9418
-2604 6056
4182 5847
-9347 9634
5549 4198
404 -2645
-6922 145
-5699 -3733
5056 3956
5432 5809
-1675 -7052
9941 -3663
4235 -8205
6151 1551
921 -8332
-6831 4143
-5601 1684
-2981 -9992
-474 -8408
1425 1448
7098 -2210
-9185 -5341
7076 754
8052 -1213
-6132 -2636
-2226 5453
8269 6777
9787 3765
-1584 2159
-858 4187
5075 -5741
-5600 -4013
-1149 -854
-8867 -6035
-2630 1376
1695 -1928
9713 623
-2517 2571
-598 -7653
4573 4232
3018 7084
-9619 -9231
-3441 -4595
-2929 7888
3031 -173
-7769 -3914
-5946 1140
-4084 -5726
-5538 -7141
-764 -2839
-6497 8487
3555 1740
-5933 8785
8891 5929
-3135 8123
5596 -3353
-426 -5182
-8966 2139
720 -175
-8336 -3745
-7904 -2840
-6411 9394
2260 -1787
-6265 1620
-1753 3888
2042 -9476
4765 -3494
4568 2377
-4061 -2054
-2684 -4987
5251 4231
-9511 7882
-370 -8412
-6140 -7422
1824 223
-6828 9197
-4410 -3080
425 -6902
-6297 -3557
-7790 -3999
6786 -4260
-5640 8271
-1268 -6881
-3051 -5473
-7565 3126
8726 9578
-2923 -1703
4434 -4769
7784 3480
1235 2158
7689 5654
7677 452
8385 -375
-8211 -1579
-2593 -5850
-3098 -61
2527 1195
2032 -6923
-3856 -9700
7883 3888
-6628 1811
-1981 -4729
-5443 7525
-3695 -2952
-8614 -9554
-492 6018
2818 5972
368 8959
8597 4846
3117 -6471
6111 2393
9824 -5418
-5052 -7903
-4084 1027
-1617 -74
-2723 6018
8630 2573
587 -3363
6705 9877
-266 8460
-5119 9491
-2472 6388
-8331 -7170
9492 -6047
-1261 8454
5943 -9868
7198 -6255
-6656 -9478
1576 -7928
5635 4073
4530 1824
-3363 5427
5983 9274
-2884 -6923
-9490 -8920
-881 -5320
2311 -6366
-1868 -8709
-5362 -8058
-1655 408
-1245 5353
-3109 7412
-3465 -613
3595 7937
-7373 -9886
-616 -7166
-6969 6780
5425 -6049
