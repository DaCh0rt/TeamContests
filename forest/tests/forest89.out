77716.01 409060935.51
